unlock-app/package.json

```diff
@@ -8,7 +8,6 @@
     "@storybook/addon-knobs": "^4.1.13",
     "@storybook/addon-storyshots": "^4.1.13",
     "@storybook/addon-viewport": "^4.1.13",
-    "@storybook/addons": "^4.1.13",
```

Comment:
I'm surprised this is not used!

Reply:
me too! which makes me think a lot more stuff is unused!